script.js

```diff
@@ -0,0 +1,162 @@
+const toDoForm = document.querySelector(".thirdRow");
+const toDoInput = document.querySelector(".todoInput");
+const toDoList = document.querySelector(".firstList");
+const completeList = document.querySelector(".secondList");
+const count = document.querySelector(".firstCount");
+const count2 = document.querySelector(".secondCount");
```

Comment:
다른 변수 명들은 다 이해하기 쉬웠습니다! 
여기 count 부분 변수도 변수 명을 예를 들어
```suggestion
const toDoCount = document.querySelector(".firstCount");
const DoneCount = document.querySelector(".secondCount");
```
이런 식으로 해주면 더 좋지 않을까 생각이 듭니다~~!

Reply:
아하 변수명이 훨씬 보기 좋네요 감사합니다!